UserInput.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using GameOfLife;
+
+namespace GameOfLife
+{
+    public static class UserInput
+    {
+        public static int validUserInput;
+        public static string wrongUserInput = "Wrong input, enter integer value.";
+        /// <summary>
+        /// Gets the value of either Height or Width from the user.
+        /// </summary>
+        /// <param name="promptMessage"></param> Message that will be displayed to user.
+        /// <returns></returns> Returns an integer value from user input.
```

Comment:
Parameter and return comments should be written between corresponding tags
like
```
        /// <summary>
        /// Gets the value of either Height or Width from the user.
        /// </summary>
        /// <param name="promptMessage">Message that will be displayed to user.</param>
        /// <returns>Returns an integer value from user input.</returns>
````
Check other places too.

Reply:
Comments written between tags.